drift/lib/src/runtime/manager/ordering.dart

```diff
@@ -0,0 +1,96 @@
+part of 'manager.dart';
+
+/// Defines a class which is used to wrap a column to only expose ordering functions
+class ColumnOrderings<T extends Object> {
+  /// This class is a wrapper on top of the generated column class
+  ///
+  /// It's used to expose ordering functions for a column
+  ///
+  /// ```dart
+  /// extension on FilterComposer<DateTime>{
+  ///  FitlerBuilder after2000() => isAfter(DateTime(2000));
+  ///}
+  /// ```
```

Comment:
Should this example be on the filter APIs instead?

Reply:
Lol, classic copy-paste. Will make fit with ordering.